docs/contributors_guide/maintenance/iso_release_procedures/Build_ISOs_and_Deploy_simp_project.rst

```diff
@@ -0,0 +1,9 @@
+Build Final ISO and Deploy to simp-project 
+==========================================
+
+FILL-ME-IN
```

Comment:
@op-ct When only one person knows how to do this, that person becomes the single point of failure.